docs/reference/commandline/network_create.md

```diff
@@ -144,13 +144,13 @@ $ docker network create -d overlay \
 ```
 
 Be sure that your subnetworks do not overlap. If they do, the network create
-fails and Engine returns an error.
+fails and Docker Engine returns an error.
```

Comment:
here, either works, but I would've probably put "daemon" if I wrote this from scratch.